src/xdocs/google_style.xml

```diff
@@ -665,7 +665,7 @@
                             <td><a name="4.6.1"/><a href="#4.6.1"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.6.1-vertical-whitespace">4.6.1 Vertical Whitespace</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.6.1-vertical-whitespace">4.6.1 Vertical Whitespace</a>
```

Comment:
Editorial + Pointer to section 4.8.1 that describes vertical spacing in enum declarations.